docs/reference/config.md

```diff
@@ -333,13 +333,14 @@ You should not configure it unless you understand what it is for.
 
 #### markdown.code.lineNumbers
 
-- Type: `boolean`
+- Type: `boolean | number`
 
-- Default: `true`
+- Default: `4`
 
 - Details:
 
-  Enable code line numbers or not.
```

Comment:
The first line is a general description of the option, and should not be removed.